cpanfile

```diff
@@ -28,7 +28,7 @@ requires 'DateTime::Format::Duration'                 => '1.04';
 requires 'DateTime::Format::ISO8601'                  => '0.08';
 requires 'DateTime::Format::Natural'                  => '1.08';
 requires 'DateTime::Format::Pg'                       => '0.16013';
-requires 'DateTime::Locale'                           => '1.25';
+requires 'DateTime::Locale'                           => '1.35';
```

Comment:
just a note: this specifies the minimum required version, i.e. "1.25 or greater," which is why we already had DateTime-Locale-1.35 in cpanfile.snapshot -- cpanm will install the latest version that satisfies the minimum requirement

Reply:
True. That explains the inconsistencies in the list of locales between the Perl code (which currently uses the version 1.35 set in `cpanfile.snapshot`) and the Node code that relies on `locales.json` (which was synced with 1.25 only).

To prevent this issue from occurring again, I just pushed commit <https://github.com/metabrainz/musicbrainz-server/pull/2599/commits/dba5fc8bba610df8baf4ec0c292e98ae3c3197f2> to specifically require the version `1.35`.